check-your-phone.md

```diff
@@ -0,0 +1,2 @@
+You look up and suddenly, there is a woman in 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.